libnavigation-core/api/current.txt

```diff
@@ -9,11 +9,12 @@ package com.mapbox.navigation.core {
     method public com.mapbox.navigation.core.history.MapboxHistoryRecorder getHistoryRecorder();
     method public com.mapbox.navigation.core.replay.MapboxReplayer getMapboxReplayer();
     method public com.mapbox.navigation.base.options.NavigationOptions getNavigationOptions();
+    method public java.util.List<com.mapbox.navigation.base.route.NavigationRoute> getNavigationRoutes();
     method public com.mapbox.navigation.core.trip.session.NavigationSessionState getNavigationSessionState();
-    method public com.mapbox.navigation.core.reroute.RerouteController? getRerouteController();
```

Comment:
There's a new `method public com.mapbox.navigation.core.reroute.NavigationRerouteController? getRerouteController();` and since `NavigationRerouteController` also implements `RerouteController`, this is change will not break API compatibility.